consensus/ibft/signer/extra.go

```diff
@@ -62,6 +84,12 @@ func (i *IstanbulExtra) MarshalRLPWith(ar *fastrlp.Arena) *fastrlp.Value {
 	// ParentCommittedSeal
 	if i.ParentCommittedSeals != nil {
 		vv.Set(i.ParentCommittedSeals.MarshalRLPWith(ar))
+	} else if i.RoundNumber != nil {
```

Comment:
Why is there  `else if i.RoundNumber != nil` and bellow again if with same condition

Reply:
Fixed 4f5ec060938dfd3d015f7d3631f0ace119caa4a9

Reply:
Fixed 4f5ec060938dfd3d015f7d3631f0ace119caa4a9

Reply:
To put RoundNumber in correct position

Reply:
I understand, resolved